### PR TITLE
Adds import of the prebuild SDK via cocoapods

### PR DESCRIPTION
**Pull request by igorkravchenko**

### CODE REVIEW

File: FitpaySDK.podspec [hunk 1]
```diff
@@ -6,16 +6,23 @@ Pod::Spec.new do |s|
   s.version = '0.1.0'
   s.license = 'MIT'
   s.summary = 'Swift based library for the Fitpay Platform'
-  s.homepage = 'https://github.com/fitpay/fitpay-ios-sdk'
+  s.homepage = 'https://bitbucket.org/oanton/fitly'
```
Comment (ssteveli) [inline FitpaySDK.podspec]: I suspect this was changed by mistake?

**Comment by ssteveli** [comment]
Is this standard to have a compiled artifact/output like this in an iOS SDK?

File: FitpaySDK.podspec (hunk 1 above)
Comment (igorkravchenko) [inline FitpaySDK.podspec]: It is changed by mistake. I'll change this back.

**Comment by igorkravchenko** [comment]
I suppose the standard way would be using of pure Swift implementation sources, but unfortunately some encryption/decryption algorithms aren't available in Swift.  
In order to ship SDK as pod it has to be precompiled first because of static OpenSSL binaries.

**Pull request merged**